lisa/transformers/kernel_source_installer.py

```diff
@@ -137,7 +137,9 @@ def _install_build(self, node: Node, code_path: PurePath) -> None:
         make = node.tools[Make]
         make.make(arguments="modules", cwd=code_path, sudo=True)
 
-        make.make(arguments="modules_install", cwd=code_path, sudo=True)
+        make.make(
```

Comment:
When is it needed? Does it affect other usage? Please explain and add into commit description.

Reply:
The Linux Next and Linux Stable builds are using this transformer to install kernel from source. It results in very large initrd image.
Should this be changed to a parameter for the kernel source installer transformer instead of statically adding it to modules_install?

Reply:
If there is no side effect of this change, it can be static here.

Reply:
Updated commit